doc/rst/language/spec/methods.rst

```diff
@@ -426,6 +428,113 @@ resolution (see :ref:`Determining_Most_Specific_Functions`).
 
       221
 
+.. index::
+   single: methods; operators
+   single: operators; methods
+.. _Operator_Methods:
+
+Operator Methods
+----------------
+
+Operators may be overloaded to support new behavior on one or more types using
+the ``operator`` keyword.  Such overloads may be defined as standalone
+functions, e.g.
```

Comment:
Probably a good idea to link to the rst location for https://chapel-lang.org/docs/language/spec/procedures.html#function-and-operator-overloading since that defines what operators are allowed